web/source/utils/version.ts

```diff
@@ -9,6 +9,9 @@ namespace com.keyman.utils {
     // as it results in unexpected, bug-like behavior for keyboard designers when it is unwanted.
     public static readonly NO_DEFAULT_KEYCAPS = new Version([12, 0]);
 
+    // Corresponds to the most recent known version of KMW and Keyman Developer.
+    public static readonly CURRENT = new Version([12, 0]);
```

Comment:
I'm a little leery of adding a new version number that we'll need to update with each version. Can we source this from resources/VERSION.md?

Reply:
Oh right, KMW has this already in a string on the base object (via "environment.inc.ts").  I'll just use that instead.